src/server/appserver.cpp

```diff
@@ -3022,6 +3025,24 @@ bool AppServer::initLogging() noexcept {
                                .c_str());
     return true;
 }
+void AppServer::logUsefulInformation() const {
+    // Log app ID
```

Comment:
It might be interesting to consider moving other useful information here (such as extended logs).

Reply:
Done in d12235a68d1f80b5fdbc84aee029baafac34ebcd